internal/handlers/pg/pgdb/settings.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pgdb
+
+import (
+	"context"
+	"fmt"
+	"hash/fnv"
+
+	"github.com/jackc/pgx/v4"
+	"golang.org/x/exp/slices"
+
+	"github.com/FerretDB/FerretDB/internal/fjson"
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+)
+
+const (
+	// Internal collections prefix.
+	collectionPrefix = "_ferretdb_"
+
+	// Settings table name.
```

Comment:
Hm, I though it depends on collection name. But now I see it's a FerretDB - database wide settings table name? 
But you replace the settings value each time...

Reply:
Yes, it's FerretDB database settings table name.